crates/test-utils/Cargo.toml

```diff
@@ -33,6 +33,7 @@ rand.workspace = true
 snapbox = { version = "0.6", features = ["json", "regex", "term-svg"] }
 tempfile.workspace = true
 ui_test = "0.30.2"
+portable-pty = "0.9.0"
```

Comment:
I would prefer we avoid introducing a new dependency just for this test and merge without a strict unit test given the behavior of https://docs.rs/comfy-table/latest/comfy_table/enum.ContentArrangement.html#variant.Dynamic is well defined and observable 

Apologies for the inconvenience

Other than that the PR look good to me :+1: !